### ISSUE TITLE
Oct 19 - 23 - Take Your First Steps with Python 

### ISSUE THREAD

**Issue by mebaumb** [microsoft/ReactorPlanning#112]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Coding, Languages, and Frameworks |
| Monthly Topic (if applicable) | Open Source |
| Weekly Topic (if applicable) | Python  |
| Workshop Duration | 60 mins | 
| Title | Take Your First Steps with Python |
| Description | <p>Are you interested in learning how to code with Python? This one-hour workshop will teach you how Python is typically used and how to set up your development environment with Visual Studio Code. You will write your first Python programs and walk away with the resources to continue learning on your own.</p><p>Prerequisites: This workshop recommends that you have a basic knowledge of a programming language other than Python. To follow along, install Python (https://www.python.org/downloads/), Visual Studio Code (https://code.visualstudio.com/Download), and create an Azure account ((https://azure.microsoft.com/free/).</p> |
| Learn Content | https://docs.microsoft.com/learn/paths/python-first-steps/ (Modules 1-3) |

# Workshop(s) details
| Date |  Local Start Time |  Location |  Speaker | SharePoint ID | Platform |
|------|-------------------|-----------|----------|---------------|----------|
| Oct 20 | 6:30 PM | Bangalore | TBD | 11635 |TBD|
| Oct 20 | 4:00 PM PST | US/Canada/Sydney | Phil Krull (Todd Enders - Moderator) | 11637 |TBD|
| Oct 21 | 7:30 PM | Sao Paulo | TBD | 11643 | TBD |
| Oct 21 | 7:30 PM | Shanghai | TBD | 11644 | TBD |
| Oct 21 | 5:00 PM | London, Stockholm, Tel Aviv, Abu Dhabi | TBD | 11645 | TBD |

# Event Checklist

## 10 Weeks Prior
- [x] Reactor Content Team (RCT) opens a new issue for an upcoming local workshop filling in the Event Information and Workshop details tables. RCT member who owns content (content owner) this week will be the assignee.
- [x] RCT and content owner discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _local-digital-event_ tag to the issue and adds event to the __Local Digital Events__ project. Issue will appear in the __Upcoming Streams__ column
- [x] RCT adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [ ] RCT informs the Reactor Marketing Team of the new event information
- [ ] RCT confirms the speaker
- [ ] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT moves the issue to the __Content Preparation In Progress__ column

## 5 weeks Prior
- [ ] Content owner opens a PR with the new content for the workshop on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors/tree/main/online-event-resources) and adds the _content-development_ and _local-digital-event_ tag to the PR. 
- [ ] RCT reviews, approves, and merges the changes
- [ ] RCT shares content with the vendor/instructor
- [ ] RCT provides vendor/instructor with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT ensures that instructors have received an invite to the workshops and that PM team works with instructors to get setup in the platform (e.g. Teams, Teams Live Event)

## 1 Week Prior
- [ ] Connect vendor/instructor with the PM(s) for the locations where this event has been marketed
- [ ] Check-in with vendor/instructor that they are prepared for the workshop
- [ ] RCT moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Instructor (and optional moderator) run the stream
- [ ] _(Optional)_RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by mebaumb** [comment]
@KimberlyBolton1 in case you didn't see this one - there is another one :) 

These have been entered into Dynamics and just need to be updated when final details come through.

**Comment by KimberlyBolton1** [comment]
@meaghanlewis  Hi Meaghan, here's the edit. Can you give me some prereqs to include--knowledge level and what they should have installed before they come? I took a stab but need your help.

**Take Your First Steps with Python**
Are you interested in learning how to code with Python? This one-hour workshop will teach you how Python is typically used and how to set up your development environment with Visual Studio Code. You will write your first Python programs and walk away with the resources to continue learning on your own.
Prerequisites: This workshop recommends that you have a basic knowledge of a programming language other than Python. To follow along, install  Python (https://www.python.org/downloads/), Visual Studio Code (https://code.visualstudio.com/Download), and create an Azure account ((https://azure.microsoft.com/free/).

**Comment by meaghanlewis** [comment]
Thank you @KimberlyBolton1! ✅

**Comment by meaghanlewis** [comment]
Here's an overview of what this workshop will cover and suggested agenda to follow with links and resources.

## Content to teach
This workshop will take Reactor community members through the first three modules of
- [Take your first steps with Python](https://aka.ms/FirstStepsWithPython-5)

**This link should be shared with attendees during the workshop:**
- https://aka.ms/FirstStepsWithPython-8

Encourage attendees to go through the modules with you and complete each one so that they get acheivement badges in their profile.


## Workshop Steps
This content starts at a very high level, so set expectations with attendees that this is truly an introduction to Python.
1. [What is Python](https://docs.microsoft.com/learn/modules/python-introduction//?WT.mc_id=reactor-github-reactor)
    - Show: https://www.python.org/
2. [Set up your Python development environment](https://docs.microsoft.com/learn/modules/python-install-vscode//?WT.mc_id=reactor-github-reactor)
    - Encourage everyone to download the following:
      - https://www.python.org/downloads/
      - https://code.visualstudio.com
      - https://marketplace.visualstudio.com/items?itemName=ms-python.python
3. [Create your first program](https://docs.microsoft.com/learn/modules/python-create-first//?WT.mc_id=reactor-github-reactor)
    - Make sure you encourage attendees to go thrugh the Challenge in Unit 5. You can also provide additional challenges for them to try.
4. Next steps
    - Encourage attendees to complete the other modules in the path: https://aka.ms/FirstStepsWithPython-8
4. Closing
    - Q&A
    - Survey: https://aka.ms/Reactor/Survey
    - Share Resources 
      - Note: The event code is listed in this issue description and/or your Reactor contact can share that with the community members
    - Check if the Reactor contact has other information to share

**Comment by KimberlyBolton1** [comment]
@mebaumb abstract final and moved to Ready for SharePoint.

**Comment by mebaumb** [comment]
Title/Abstract sent to Caio for translation.

US/Canada - Published to Dynamics

All other locations - updated, will publish when I have speaker info.

**Comment by mebaumb** [comment]
Bangalore, Sao Paulo, Shanghai, and London/Stockholm/Abu Dhabi/Tel Aviv are all published without speaker data (9/2)

**Comment by mebaumb** [comment]
Events delivered and attendance data recorded. Closing issue.
